harold/plugins/deploy.py

```diff
@@ -41,25 +42,17 @@ class DeployConfig(PluginConfig):
     blackout_hours_end = Option(parse_time)
 
 
+class DeployHoldType(Enum):
+    code_freeze = 'Code Freeze'
+    manual = 'Manual'
+
+
 class DeployListener(ProtectedResource):
     def __init__(self, http, monitor):
         ProtectedResource.__init__(self, http)
         self.monitor = monitor
 
 
-class DeployGetSalonNamesListener(DeployListener):
```

Comment:
Moved here so that the `DeployListener` types are all together.
https://github.com/spladug/harold/pull/40/files#diff-270925a10931db85f0b8727a98cef1edR229